mk/docs.mk

```
@@ -1,30 +1,34 @@
-DESTDIR := docs/generated
+DOCS_PROTOS ?= api/mesh/v1alpha1/*.proto
+DOCS_CP_CONFIG ?= pkg/config/app/kuma-cp/kuma-cp.defaults.yaml
+
+.PHONY: clean/docs
+clean/docs:
+	rm -rf docs/generated
 
 .PHONY: docs
-docs: helm-docs ## Dev: Generate local documentation
-	rm -rf $(DESTDIR)
-	@$(MAKE) docs/install/markdown DESTDIR=$(DESTDIR)/cmd
-	@$(MAKE) docs/install/resources DESTDIR=$(DESTDIR)/resources
+docs: docs/generated/cmd docs/generated/kuma-cp.md docs/generated/resources helm-docs ## Dev: Generate local documentation
 
 .PHONY: helm-docs
 helm-docs: ## Dev: Runs helm-docs generator
 	$(HELM_DOCS) -s="file" --chart-search-root=./deployments/charts
 
-.PHONY: docs/install/markdown
-docs/install/markdown: DESTDIR:=$(DESTDIR)/markdown
-docs/install/markdown: ## Generate CLI reference in markdown format
-	@DESTDIR=$(DESTDIR) FORMAT=markdown go run $(TOOLS_DIR)/docs/generate.go
-
-.PHONY: docs/install/manpages
-docs/install/manpages: DESTDIR:=$(DESTDIR)/manpages
-docs/install/manpages: ## Generate CLI reference in man(1) format
-	@DESTDIR=$(DESTDIR) FORMAT=man go run $(TOOLS_DIR)/docs/generate.go
+DOCS_CMD_FORMAT=markdown
```

I know we need markdown here, but maybe for futureproofing we could go with `?=` 
```suggestion
DOCS_CMD_FORMAT ?= markdown
```